The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok

WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start

waiting for server to start....2025-02-14 13:14:28.893 UTC [46] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:14:28.904 UTC [47] LOG:  database system was shut down at 2025-02-14 13:14:28 UTC
2025-02-14 13:14:28.908 UTC [46] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 13:14:29.262 UTC [46] LOG:  received fast shutdown request
2025-02-14 13:14:29.263 UTC [46] LOG:  aborting any active transactions
2025-02-14 13:14:29.265 UTC [46] LOG:  background worker "logical replication launcher" (PID 53) exited with exit code 1
2025-02-14 13:14:29.266 UTC [48] LOG:  shutting down
2025-02-14 13:14:29.276 UTC [46] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 13:14:29.370 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 13:14:29.370 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 13:14:29.372 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:14:29.384 UTC [64] LOG:  database system was shut down at 2025-02-14 13:14:29 UTC
2025-02-14 13:14:29.387 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 13:14:36.086 UTC [79] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 13:14:36.086 UTC [79] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 13:14:36.432 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:14:36.432 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_fe56aee6-c019-4677-a488-3f99b33823c7" WITH (FORCE)
2025-02-14 13:14:36.977 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:14:36.977 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_e274a261-1d90-49c1-aad5-41b4a59e6d58" WITH (FORCE)
2025-02-14 13:14:37.491 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:14:37.491 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_90eecafc-724d-408e-af99-0f3b5db5b92c" WITH (FORCE)
2025-02-14 13:14:38.045 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:14:38.045 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_7b1ac169-f149-4937-a5c7-218958e6ca85" WITH (FORCE)
